### ISSUE TITLE
IOU reward not converge

### ISSUE THREAD

**Issue by LaFeuilleMorte** [Liuziyu77/Visual-RFT#14]
Hi, I've tried GRPO on your coco_base65 dataset with Qwen2.5-VL-3B-Instruct base model. But after training for several hundreds of steps, the IOU reward does not converge:

<img width="1382" alt="Image" src="https://github.com/user-attachments/assets/01fffa8c-6c83-4c38-aa8f-b0d3bedb94d3" />

my training script:

cd src/virft

export DEBUG_MODE="false"
export LOG_PATH="./debug_log_2b_GRPO_coco_base65cate_6k.txt"

export RUN_NAME=Qwen2.5-VL-3B-Instruct_GRPO_coco_base65cate_6k 
export DATA_PATH=/xxx/Visual-RFT/datasets/laolao77/ViRFT_COCO_base65  ### your local dataset downloading from huggingface
export CKPT_PATH=/xxx/Visual-RFT/checkpoint/Qwen/Qwen2.5-VL-3B-Instruct   ### Qwen2-VL-2B checkpoint path
export SAVE_PATH=/xxx/Visual-RFT/output/models/${RUN_NAME}   ### save path

torchrun --nproc_per_node="8" \
    --nnodes="1" \
    --node_rank="0" \
    --master_addr="127.0.0.1" \
    --master_port="12345" \
    src/open_r1/grpo.py \
    --output_dir ${SAVE_PATH}  \
    --model_name_or_path ${CKPT_PATH} \
    --dataset_name ${DATA_PATH} \
    --deepspeed local_scripts/zero3.json \
    --max_prompt_length 1024 \
    --per_device_train_batch_size 1 \
    --gradient_accumulation_steps 2 \
    --logging_steps 1 \
    --bf16 \
    --report_to wandb \
    --gradient_checkpointing false \
    --attn_implementation flash_attention_2 \
    --max_pixels 401408 \
    --num_train_epochs 10 \
    --run_name ${RUN_NAME}  \
    --save_steps 100 \
    --save_only_model true \
    --num_generations 4

**Comment by Liuziyu77** [comment]
Qwen2-VL outputs relative coordinates normalized to [0, 1000], while Qwen2.5-VL uses absolute coordinates. Therefore, modifications to the training data and IoU reward calculation are needed to adapt to Qwen2.5-VL's detection tasks. We will support Qwen2.5-VL as soon as possible.

**Comment by LaFeuilleMorte** [comment]
> Qwen2-VL outputs relative coordinates normalized to [0, 1000], while Qwen2.5-VL uses absolute coordinates. Therefore, modifications to the training data and IoU reward calculation are needed to adapt to Qwen2.5-VL's detection tasks. We will support Qwen2.5-VL as soon as possible.

Thank you so much for the reply!
